install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml

```diff
@@ -513,8 +513,6 @@ spec:
                           volumeCapacity:
                             type: string
                         type: object
-                      rules:
```

Comment:
Is this change required by this PR scope?

Reply:
This CRD file is automatically generated by operator-sdk when the `install/operator/build.sh` runs. The property was removed in this PR https://github.com/syndesisio/syndesis/pull/8979 but was not pushed. This should be pushed anyway.